src/app/design/guidelines/error-handling/index.html

```diff
@@ -27,6 +27,104 @@
 	<p>
 		Don't rely on a single sensory characteristic of components such as shape, size, visual location, orientation, or sound to describe an error or offer a solution. "Above" and "below" are fine for referencing content in the appropriate reading order for the language used as long as the reference isn't ambiguous.
 	</p>
+
+	<stache-page-anchor>
+		Error message images
+	</stache-page-anchor>
+
+ 	<p>
+		The error message images apply to different scenarios and should be used accordingly. 
+	</p>
+
+ 	<sky-repeater expandMode="multiple">
+		<sky-repeater-item [isExpanded]="false">
+			<sky-repeater-item-title>Broken (Bird)</sky-repeater-item-title>
+			<sky-repeater-item-content>
+				<stache-image
+        		imageSource="~/assets/img/guidelines/errorhandling/bird.png">
+       		 	</stache-image>
+
+      			<p>
+					Use this image for:
+				</p>
+        		<ul>
+            		<li>Errors with the modal component. Example: Modal can't launch form, or the modal service is down on save.</li>
+            		<li>Errors loading list data. Example: Tab content doesn't load.
+            		<li>Errors with the list component. Example: List contents don't load.
```

Comment:
Seems like this should be "List content doesn't" to match the previous item or that the previous item should be "Tab contents don't" to match this. Switching seems weird.

Reply:
Also, I'd pull the examples into the preceding sentences with a "such as when" as described above.